extras/docker/fromsource/Dockerfile

```diff
@@ -11,6 +11,8 @@ ENV GOPATH=$BUILD_HOME/golang
 ENV PATH=$GOPATH/bin:$PATH
 ENV HEKETI_BRANCH="master"
 
+RUN curl https://glide.sh/get | sh
+
 # install dependencies, build and cleanup
 RUN mkdir $BUILD_HOME $GOPATH && \
```

Comment:
This is not a big change but we can reduce a layer.

RUN curl https://glide.sh/get | sh && \
mkdir $BUILD_HOME $GOPATH && \ 
.
.
.